catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/campaign_status/campaign_status_cubit.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:meta/meta.dart';
+
+part 'campaign_status_state.dart';
+
+class CampaignStatusCubit extends Cubit<CampaignStatusState> {
```

Comment:
Probably a bloc/cubit that manages the whole campaign (not only it's status) will be needed. I would design this class in a way that it can handle all the operations/properties of a campaign (in context of editing it).

For now, just rename to `CampaignCubit`?

In the future we might need to change it to `CampaignBuilderCubit` etc as it would have operations specific to creating a campaign